### DISCUSSION TITLE
Error uploading large files to MinIO

### DISCUSSION THREAD

**Discussion by rex232** [appany/Minio.AspNetCore#38] [General]
![image](https://user-images.githubusercontent.com/1892642/146472343-0a377894-0854-4bb7-840f-74c57755ebbe.png)
Hi, I have a problem on upload  a large file to minio, hope you can help.

This function is web API written in ASP.NET Core running in linux docker, when the file size is over 5MB, the UploadFile will failed and without any exception. 

The api controller header is like this
[HttpPost, DisableRequestSizeLimit]
I had added DisableRequestSizeLimit but seem not working, any advise? many thx

**Comment by sergeyshaykhullin** [comment]
This is repo for Minio + AspNetCore integration, seems like your problem is about Minio.NET itself. Please file an issue in https://github.com/minio/minio-dotnet with `UploadFile` contents provided
